### ISSUE TITLE
[CascaderSelect]Let Cascader Select component support setting focus through the "focus" method

### ISSUE THREAD

**Issue by Mikawawawa** [alibaba-fusion/next#3608]
### Component 
CascaderSelect
### Feature Description
I am adding shortcut key support to my form, so I need to control the focus of the control in the logic code. I noticed that the Input component has a focus method, and it is useful. However, I cannot control the focus of the cascader component
I think that similar form controls should add focus and blur methods

**Comment by lancelotnemoj** [comment]
打扰了，我以为大家都是提的英文issue，我现在在给一个表单加键盘设置，fusion已有的tab交互和我们的实际场景不太一致，我们需要快速在控件中跳转光标，实际需要输入的时候再按一个键来开启编辑，这样的话，我就要再 keydown 事件的处理逻辑中指定控件被focus，目前input是可以的，但是cascader不太行，我先试下自己包一层来解决

**Comment by YSMJ1994** [comment]
近期版本我们将会给 cascader-select 添加 focus 方法，并使其可通过键盘操控下拉框展开 & 选择
